icu4c/source/i18n/measunit_extra.cpp

```diff
@@ -483,7 +486,7 @@ class Token {
 
   static Token constantToken(StringPiece str, UErrorCode &status) {
       Token result;
-      auto value = result.parseStrigToLong(str, status);
+      auto value = Token::parseStrigToLong(str, status);
```

Comment:
You still have `parseStrigToLong()` instead of `parseStriNgToLong()`.

Reply:
done, thanks for the note :)